### DISCUSSION TITLE
Accessing SVN "Versioned Properties"

### DISCUSSION THREAD

**Discussion by DynConcepts** [AmpScm/SharpSvn#94] [Q&A]
I can not figure out how to do this....  Even the SVN API docs are not helpful (if they were, I could probably wade in...

Any help appreciated.

https://svnbook.red-bean.com/en/1.7/svn.ref.properties.html

**Comment by rhuijben** [comment]
There are two types of properties. Versioned properties are attached to a node (read: a file or a directory) and revision properties that are just attached to a revision and (if hooks allow it) may be changed later.
